Fix efferent broken dependency with failure v1.0.6 manifesting in vscode dev containers

<!-- Thanks for filing a 🐛 bug report 😄! -->

**Problem**

Cargo fails to build in a vscode dev container:

```bash
   Compiling ahash v0.2.18
error[E0433]: failed to resolve: could not find `__rt` in `quote`
   --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/failure_derive-0.1.6/src/lib.rs:107:70
    |
107 | fn display_body(s: &synstructure::Structure) -> Result<Option<quote::__rt::TokenStream>, Error> {
    |                                                                      ^^^^ could not find `__rt` in `quote`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0433`.
```

This is a known bug with the failure crate `v1.0.6`:

https://github.com/rust-lang-nursery/failure/issues/342

**Steps**

1. Create a dev container instance in vscode
2. Run cargo build
3. The build fails with the above error

**Possible Solution(s)**

Pin the `quote` crate to v1.0.2

**Notes**

Output of `rustup --version`:

```
rustup 1.21.1 (7832b2ebe 2019-12-20)
```

Output of `rustup show`:

```
active toolchain
----------------

stable-x86_64-apple-darwin (default)
rustc 1.41.0 (5e1a79984 2020-01-27)
```

Output of `tremor-server --version`:

Does not build due to cargo build error in this issue

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -98,6 +98,10 @@ grok = "1"
 
 snmalloc-rs = { version = "0.2", optional = true }
 
+# Pin quote crate to resolve https://github.com/rust-lang-nursery/failure/issues/342
+# This impacts failure-derive v1.0.6 and manifests in vscode dev container
+quote = "=1.0.2"
+
 [dependencies.tungstenite]
 version = "0.10"
 default-features = false

